native/dlartg.go

```diff
@@ -17,7 +17,8 @@ import "math"
 func (impl Implementation) Dlartg(f, g float64) (cs, sn, r float64) {
 	safmin := dlamchS
 	eps := dlamchE
-	safmn2 := math.Pow(float64(dlamchB), math.Log(safmin/eps))
+	exp := int(math.Log(safmin/eps) / math.Log(float64(dlamchB)) / 2)
+	safmn2 := math.Pow(float64(dlamchB), float64(exp))
```

Comment:
`dlamchB` is an int variable but yes, it would be nice if it were a const.
I don't know where else this value is used to make it package global. Unfortunately, this kind of floating point manipulation is mostly magic to me.